.github/workflows/ci.yml

```diff
@@ -16,65 +16,70 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 20
     steps:
-    - uses: actions/checkout@v3
-
-    - name: Install Docker Compose via Package Manager
-      run: |
-        sudo apt-get update
-        sudo apt-get install -y docker-compose
-      shell: bash
-
-    - name: Verify Docker Compose Installation
-      run: docker-compose --version
-      shell: bash
-
-    - name: Start a private ethereum network
-      uses: ./.github/actions/geth
-      id: geth
-
-    - name: Sleep for 20 seconds
-      run: sleep 20s
-      shell: bash
-
-    - name: Get latest block from geth node
-      run: |
-        curl -X POST "http://127.0.0.1:8546" --header 'Content-Type: application/json' --data '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["latest", true],"id":1}'
-      shell: bash
-
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install web3
-
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install web3==5.31.3
-      shell: bash
-
-    - name: deploy erc20 USDC
-      run: .github/scripts/init_erc20.sh
-      shell: bash
-
-    - name: Get erc20 infos
-      run: python .github/scripts/contract_infos.py
-      shell: bash
-
-    - name: Populate transactions
-      run: python .github/scripts/populate_txns.py
-      shell: bash
-
-    - name: Start Rosetta Server
-      run: .github/scripts/setup.sh
-      shell: bash
-
-    - name: Run Check:construction test
-      run: .github/scripts/construction.sh
-      shell: bash
-
-    - name: Run Check:data test
-      run: .github/scripts/cli.sh
-      shell: bash
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
```

Comment:
All other changes are just auto-format except this Setup Python workflow. We need to downgrade to 3.9 to avoid the error when importing web3 library: https://github.com/ethereum/web3.py/issues/2704#issuecomment-1369041219